atlassian_jwt_auth/contrib/flask_app/decorators.py

```diff
@@ -0,0 +1,30 @@
+from functools import wraps
+
+from flask import Response, current_app, request
+
+import atlassian_jwt_auth
+from atlassian_jwt_auth.contrib.flask_app.utils import parse_jwt
```

Comment:
You could use a `.` relative import instead here instead
`from .utils import parse_jwt`